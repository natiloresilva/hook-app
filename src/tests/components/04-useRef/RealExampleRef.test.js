import { shallow } from "enzyme"
import { RealExampleRef } from "../../../components/04-useRef/RealExampleRef"

describe('Pruebas en <RealExampleRef/>', () => {

    const wrapper = shallow(<RealExampleRef />)

    test('Debe mostrarse correctamente el componente', () => {

        expect(wrapper).toMatchSnapshot()
        expect(wrapper.find('MultipleCustomHooks').exists()).toBeFalsy
    })

    test('Debe de mostrar el componente <MultipleCustomHooks/>', () => {

        wrapper.find('button').simulate('click')
        expect(wrapper.find('MultipleCustomHooks').exists()).toBeTruthy

    })


})
