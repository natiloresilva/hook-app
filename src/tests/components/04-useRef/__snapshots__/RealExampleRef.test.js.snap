// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <RealExampleRef/> Debe mostrarse correctamente el componente 1`] = `
<Fragment>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <button
    className="btn btn-outline-info m-5"
    onClick={[Function]}
  >
    Show / Hide
  </button>
</Fragment>
`;
